docs/tools/vdb_table/data/usearch.json

```diff
@@ -108,22 +108,22 @@
     "comment": ""
   },
   "pricing": {
-    "value": "",
+    "value": "none",
     "source_url": "",
     "comment": ""
   },
   "in_process": {
-    "support": "none",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "multi_tenancy": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "disk_index": {
-    "support": "",
+    "support": "partial",
```

Comment:
reference needed

Reply:
https://github.com/unum-cloud/usearch/?tab=readme-ov-file#serialization--serving-index-from-disk

Reply:
This looks good. Please add this link to the source_url field